src/__tests_/MissionTable.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { create } from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import { render, screen } from '@testing-library/react';
+import store from '../redux/configureStore';
+import MissionTable from '../components/missions/MissionTable';
+
```

Comment:
- [ ] There are some failing test cases. Kindly ensure that **all** test cases are passing.

**Failing test:**
![image](https://user-images.githubusercontent.com/12745474/195993662-03620845-79c5-485e-900e-c60668915a0d.png)